locales/en/messages.json

```diff
@@ -559,7 +559,7 @@
         "message": "<a href=\"https://github.com/Betaflight\" target=\"_blank\">GitHub</a>"
     },
     "defaultSupport5": {
-        "message": "<a href=\"http://betaflight.tk/slack\" target=\"_blank\">Betaflight devs on slack</a>"
+        "message": "<a href=\"https://slack.betaflight.com\" target=\"_blank\">Betaflight devs on slack</a>"
```

Comment:
Should this be renamed to indicate that this is only the registration? (Or else we should add a link to the page to direct registered users to the slack login page.)

Reply:
I thought the same and I decided that is better to add the registration, and if possible add a link in the registration page to the real slack chat. 
But I'm open to change it.

Reply:
We can always change it if changing the registration page isn't an option.